app/src/pages/Play/index.tsx

```diff
@@ -119,6 +119,29 @@ function Play() {
     return null
   }
 
+  const numCompletedCards = completedCardIds.length
+  const totalNumCards = currentGame.cards.length
+  const numRounds = currentGame.rounds.length
+
+  if (numCompletedCards === numRounds * totalNumCards) {
+    return <NoMoreRounds />
+  }
+
+  const roundMarkers = [...Array(numRounds).keys()]
+  let roundMarker = Math.floor(numCompletedCards / totalNumCards)
+
+  const currentRoundId = currentGame.rounds[roundMarker].id
+  let nextRoundId
+  if (currentGame.rounds.length > roundMarker + 1) {
+    nextRoundId = currentGame.rounds[roundMarker + 1].id
+  }
```

Comment:
const nextRoundId = currentGame.rounds[roundMarker + 1]?.id

might also work for you. we should also update the Props type for nextRoundId to be undefined (with a ?)